modules/risks-setting-higher-process-id-limits.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-configuring-pid-limits.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="risks-setting-higher-process-id-limits_{context}"]
+= Risks of setting higher process ID limits for {product-title} pods
+
+The `podPidsLimit` parameter for a pod controls the maximum number of processes and threads that can run simultaneously in that pod.
+
+You can increase the value for `podPidsLimit` from the default of 4,096 to a maximum of 16,384. Changing this value may incur downtime for applications, because changing the `podPidsLimit` requires rebooting the affected node.
+
+If you are running a large number of pods per node, and you have a high `podPidsLimit` value on your nodes, you risk exceeding the PID maximum for the node.
+
+To find the maximum number of pods that you can run simultaneously on a single node without exceeding the PID maximum for the node, divide 3,650,000 by your `podPidsLimit` value. For example, if your `podPidsLimit` value is 1,048,576, and you expect the pods to use close to that number of process IDs, you can safely run three pods on a single node.
+
+[NOTE]
+====
+Memory, CPU, and available storage can also limit the maximum number of pods that can run simultaneously, even when `podPidsLimit` is set appropriately. For more information, see "Planning your environment" and "Limits and scalability".
```

Comment:
@shipsing Our guidance on this could be clarified for sure, but the ISG suggests that we use double-quotation marks: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting. The table is kind of hard to browse, but it mentions that "Topic titles in IBM Documentation" should be highlighted in double-quotation marks.

So, absent any other guidance, I think the quotation marks can be left as-is.

Reply:
Thanks for the verification - it's certainly an awkward presentation, but there's no good way around the "no xrefs in modules" rule.